From c303b7c3723f3e9fbc43963a62237ea17516fc6b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Luk=C3=A1=C5=A1=20Hr=C3=A1zk=C3=BD?= <lhrazky@redhat.com>
Date: Thu, 17 Feb 2022 18:30:14 +0100
Subject: [PATCH 34/34] libdnf/transaction/TransactionItem: Set short action
 for Reason Change

Sets the "short" (one letter) representation of the Reason Change action
to "C".

This was likely not ever used before as the only way to create a
transaction with a reason change and something else is rolling back
multiple transactions, which was broken.
---
 libdnf/transaction/TransactionItem.cpp | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/libdnf/transaction/TransactionItem.cpp b/libdnf/transaction/TransactionItem.cpp
index 3b43d1f1..4358038e 100644
--- a/libdnf/transaction/TransactionItem.cpp
+++ b/libdnf/transaction/TransactionItem.cpp
@@ -51,8 +51,7 @@ static const std::map< TransactionItemAction, std::string > transactionItemActio
     {TransactionItemAction::REMOVE, "E"},
     {TransactionItemAction::REINSTALL, "R"},
     {TransactionItemAction::REINSTALLED, "R"},
-    // TODO: replace "?" with something better
-    {TransactionItemAction::REASON_CHANGE, "?"},
+    {TransactionItemAction::REASON_CHANGE, "C"},
 };
 
 /*
-- 
2.31.1

